app/scripts/controllers/transactions/index.js

```diff
@@ -1621,6 +1621,7 @@ export default class TransactionController extends EventEmitter {
       this._trackTransactionMetricsEvent(
         txMeta,
         TRANSACTION_EVENTS.FINALIZED,
+        undefined,
```

Comment:
nit: I recommend using null instead of undefined since we are setting the value 

ref: https://stackoverflow.com/questions/5076944/what-is-the-difference-between-null-and-undefined-in-javascript

Reply:
Hey @digiwand : here I do not intent to set value but indicate absence of value. I am not sure how null will be better in this case. Can you plz share your view.

Reply:
I think you're right. Since we're not declaring any variable this is fine